application/tests/org.openjdk.jmc.jolokia.test/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,17 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: RJMX Test
+Bundle-SymbolicName: org.openjdk.jmc.jolokia.test;singleton:=true
+Bundle-Version: 9.0.0.qualifier
+Bundle-Vendor: Oracle Corporation
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
```

Comment:
Should be JavaSE-17?

Reply:
0fc546a02a936430700483d29503dad4505d1b64